editor/components/block-list/block.js

```diff
@@ -433,7 +433,7 @@ export class BlockListBlock extends Component {
 			>
 				<BlockDropZone index={ order } />
 				{ ( showUI || isHovered ) && <BlockMover uids={ [ block.uid ] } /> }
-				{ ( showUI || isHovered ) && <BlockSettingsMenu uids={ [ block.uid ] } /> }
+				{ ( showUI || isHovered ) && <BlockSettingsMenu uids={ [ block.uid ] } renderBlockMenu={ renderBlockMenu } /> }
```

Comment:
This change with `renderBlockMenu` broke the multi-select controls dropdown:

1. Create a multi-selection range
2. Open "More Options" menu

![multi-select missing](https://user-images.githubusercontent.com/1779930/35481390-98fec504-041a-11e8-97d1-9ddfe0f8c6df.png)

Reply:
Good catch, we should cherry-pick the fix from the templates branch to its own PR